src/system/dmod_system.c

```diff
@@ -170,6 +183,7 @@ Dmod_Context_t* Dmod_LoadFile( const char* Path )
     Dmod_Event_ModuleLoadingInProgress( Path, 100 );
 
     DMOD_LOG_INFO("Module loaded: %s\n", Dmod_Context_GetModuleName( context ));
+    DMOD_LOG_INFO("To debug this module in gdb: add-symbol-file %s %p\n", Path, DMOD_GET_TEXT_SECTION_ADDR(context));
```

Comment:
Tutaj to nie jest ścieżka do elf tylko do dmf niestety - i to w innym systemie plików, więc user musi sam wiedzieć gdzie to ma